components/collections/collection.vue

```diff
@@ -8,7 +8,7 @@
       @drop="dragging = false"
       @dragleave="dragging = false"
       @dragend="dragging = false"
-      @click="$emit('select-folder', { name: '', id: collection.id })"
+      @click="$emit('select-folder', { name: '', id: collection.id, reqIdx: '' })"
```

Comment:
`.row-wrapper` is used for drag and drop requests into collections and folders. I think it's better to move `@click` trigger to `<button>` component inside `.row-wrapper`.

Reply:
on it